app/models/user.rb

```diff
@@ -19,9 +19,8 @@ class User < ApplicationRecord
   # as we do not need to work with the relationship models as independent entities, `has_and_belongs_to_many` is fine
   # https://guides.rubyonrails.org/association_basics.html#choosing-between-has-many-through-and-has-and-belongs-to-many
   # rubocop:disable Rails/HasAndBelongsToMany
```

Comment:
I would move this down so that it is directly above the code lines where this cop should be disabled